geode-wan/src/main/java/org/apache/geode/cache/client/internal/locator/wan/LocatorMembershipListenerImpl.java

```diff
@@ -108,11 +109,18 @@ public void locatorJoined(final int distributedSystemId, final DistributionLocat
       final DistributionLocatorId sourceLocator) {
     // DistributionLocatorId for local locator.
     DistributionLocatorId localLocatorId;
+
+    InternalDistributedSystem system = InternalDistributedSystem.getConnectedInstance();
```

Comment:
we're trying to get rid of the use of static methods like this.  The DistributionConfig argument passed to this method should have the memberName in it.  You should use that instead.